### DISCUSSION TITLE
求助：自动翻译订阅中的所有未读条目的标题

### DISCUSSION THREAD

**Discussion by zhaojing629** [windingwind/zotero-actions-tags#260] [Q&A]
我觉得zotero的订阅功能还挺好的，可以关注一些期刊的的更新，然后通过翻译未读条目的标题快速过一下。
但是我比较困扰的是，我加了十几个期刊，每次都要点一条进去，然后选中未读的去翻译。
想知道tag有没有脚本可以实现。
我是个小白，对这些语法不太懂，所以想要问问看有没有大佬对这个感兴趣，能帮忙实现吗？

另外就是，我最开始还想过通过高级搜索功能，直接搜我关注的关键词，但是发现这个订阅还是有个问题，就是一个期刊就是一个分类，它只能在一个分类下来搜索，也无法将十几个期刊一起搜索……如果能将自己感兴趣的关键词存起来，通过tag脚本来搜索每一个期刊，感觉也会很方便。不知道能不能也实现。当然这个不是很重要，因为如果能实现前面说的那个，快速的过一遍就不用去搜索关键词了。

所以还是想问问第一个脚本有没有大佬帮帮忙。。。

**Comment by Jiang-wanlin** [comment]
同求，顺便求翻译摘要的脚本

**Reply by wzr-ms** [reply]
你的订阅能获取到摘要？

**Comment by onewood591** [comment]
同求

**Comment by wzr-ms** [comment]
同问，针对订阅的自动翻译标题的脚本
